db/migrate/20191223133115_create_pull_requests.rb

```diff
@@ -0,0 +1,18 @@
+class CreatePullRequests < ActiveRecord::Migration[6.0]
+  def change
+    create_table :pull_requests do |t|
+      t.integer :github_id
+      t.integer :number
+      t.string :state
+      t.boolean :locked
```

Comment:
avoid the three state boolean problem https://thoughtbot.com/blog/avoid-the-threestate-boolean-problem